src/xenia/app/emulator_window.cc

```diff
@@ -454,6 +454,12 @@ void EmulatorWindow::UpdateTitle() {
         ")";
   }
 
+  auto patching_system = emulator()->patching_system();
```

Comment:
auto should generally be avoided because it makes code harder to read — only for cases when you already have a cast on the right side, or for things like iterators.